src/v/raft/consensus.cc

```diff
@@ -559,10 +559,7 @@ ss::future<result<model::offset>> consensus::linearizable_barrier() {
         send_futures.push_back(std::move(f));
     });
     // release semaphore
-    // snapshot taken under the semaphore
-    state_snapshot snapshot{
```

Comment:
it seems we still use `snapshot` at line 584, it looks like this commit should not compile

Reply:
done

Reply:
nice catch